File: TransCoder_tokenized_test_set_functions/python/FIND_THE_POINT_WHERE_MAXIMUM_INTERVALS_OVERLAP.py
def findMaxGuests ( arrl, exit, n ) :
    arrl . sort ( );
    
    exit . sort ( );
    
    guests_in = 1;
    
    max_guests = 1;
    
    time = arrl [ 0 ];
    
    i = 1;
    
    j = 0;
    
    while ( i < n and j < n ) :
        if ( arrl [ i ] <= exit [ j ] ) :
            guests_in = guests_in + 1;
            
            if ( guests_in > max_guests ) :
                max_guests = guests_in;
                
                time = arrl [ i ];
                
            i = i + 1;
            
        else :
            guests_in = guests_in - 1;
            
            j = j + 1;
            
    print ( "Maximum Number of Guests =", max_guests, "at time", time )

File: TransCoder_tokenized_test_set_functions/java/FIND_THE_POINT_WHERE_MAXIMUM_INTERVALS_OVERLAP.java
static void findMaxGuests ( int arrl [ ], int exit [ ], int n ) {
  Arrays . sort ( arrl );
  Arrays . sort ( exit );
  int guests_in = 1, max_guests = 1, time = arrl [ 0 ];
  int i = 1, j = 0;
  while ( i < n && j < n ) {
    if ( arrl [ i ] <= exit [ j ] ) {
      guests_in ++;
      if ( guests_in > max_guests ) {
        max_guests = guests_in;
        time = arrl [ i ];
      }
      i ++;
    }
    else {
      guests_in --;
      j ++;
    }
  }
  System . out . println ( "Maximum Number of Guests = " + max_guests + " at time " + time );
}